generative_ai/prompts/prompt_template_delete.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+
+
+def delete_prompt_version_generate() -> str:
+    """Deletes specified prompt."""
+
+    # [START generativeaionvertexai_prompt_delete_prompt_version]
```

Comment:
Should the region marker only show the prompts.delete() call for the deletion snippet? Since right now it is showing the whole thing